Khan/wonder-blocks
# Set up Storybook and write some button stories

This is in preparation for testing out chromatic for visual regression testing of our components.  The more stories we have the more snapshots chromatic will take which will take longer during builds.  This is why some of the stories group buttons together.

Test Plan:
- yarn start:storybook
- open http://localhost:53724/
- see the button stories
- yarn build:storybook
- see that storybook-static has been created
- http-server storybook-static
- open http://localhost:8080/, see that this storybook instance serves the same content

<img width="952" alt="Screen Shot 2020-01-11 at 7 37 19 PM" src="https://user-images.githubusercontent.com/1044413/72212419-d3750380-34a9-11ea-93e0-a6069eb0504d.png">
<img width="952" alt="Screen Shot 2020-01-11 at 7 37 29 PM" src="https://user-images.githubusercontent.com/1044413/72212420-d53ec700-34a9-11ea-8088-f744d2059046.png">
